app/views/categories/index.html.erb

```diff
@@ -1,17 +1,34 @@
-<div class='category-list' itemscope itemtype='http://schema.org/ItemList'>
-  <meta itemprop='itemListOrder' content='http://schema.org/ItemListOrderDescending'>
-  <% @category_list.categories.each_with_index do |c, index| %>
-    <div class='category' itemprop='itemListElement' itemscope itemtype='http://schema.org/ListItem'>
-      <meta itemprop='position' content='<%= index %>'>
-      <meta itemprop='url' content='<%= "#{c.url}" %>'>
-      <h2>
-        <a href='<%= "#{c.url}" %>' itemprop='item'>
-          <span itemprop='name'><%= c.name %></span>
-        </a>
-      </h2>
-      <span itemprop='description'><%= c.description&.html_safe %></span>
-    </div>
-  <% end %>
+<div itemscope itemtype='http://schema.org/ItemList'>
+  <table class='category-list'>
+    <thead>
+      <tr>
+        <th class='category'><%= t 'js.categories.category' %></th>
+        <th class='topics'><%= t 'js.topic.list' %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <meta itemprop='itemListOrder' content='http://schema.org/ItemListOrderDescending'>
+      <% @category_list.categories.each_with_index do |c, index| %>
+        <tr>
+          <td class='category' style='border-color: #<%= c.color %>;'>
+            <div itemprop='itemListElement' itemscope itemtype='http://schema.org/ListItem'>
+              <meta itemprop='position' content='<%= index %>'>
+              <meta itemprop='url' content='<%= "#{c.url}" %>'>
```

Comment:
```suggestion
              <meta itemprop='url' content='<%= c.url %>'>
```

Reply:
Actually, it's an old code. I had just changed the indentation :sweat_smile:, I have fixed all your suggestions @ZogStriP